lsm-tree.cabal

```diff
@@ -289,20 +268,25 @@ test-suite lsm-tree-test
     , these
     , transformers
     , vector
+    , wide-word
 
   ghc-options:      -fno-ignore-asserts -threaded
 
 benchmark lsm-tree-micro-bench
   import:           warnings, wno-x-partial
   default-language: Haskell2010
   type:             exitcode-stdio-1.0
-  hs-source-dirs:   bench/micro
+  hs-source-dirs:   bench/micro test
```

Comment:
So now the `Extras` are build multiple times for different components, `lsm-tree-test`, `lsm-tree-micro-bench` and also `lsm-tree-macro-bench`?

Reply:
As I understood it the idea was to import the test suite, but then I realised that's not possible (or is it?). In that case, I'm not sure removing `utils` is worth it. :thinking:

Reply:
Yes the small number of modules used (out of the small number of util/extra modules that there are) would be built for each component that uses them. My view was that was a reasonable tradeoff to eliminate a whole sub-heirarchy.